pkg/controllers/allocation/bind.go

```diff
@@ -82,3 +93,55 @@ func (b *Binder) bind(ctx context.Context, node *v1.Node, pod *v1.Pod) error {
 	}
 	return nil
 }
+
+type binderMetricsDecorator struct {
+	binder               Binder
+	bindTimeHistogramVec *prometheus.HistogramVec
+}
+
+const metricLabelResult = "result"
```

Comment:
Any reason this isn't inlined?

Reply:
An old habit of avoiding bare keys in map lookups; misspellings cause runtime problems.  When using a constant, misspellings cause compile-time errors.